scripts/release/utils/getreleasedescription.mjs

```diff
@@ -20,6 +20,12 @@ export default async function getReleaseDescription( cliArguments ) {
 		];
 	}
 
+	if ( cliArguments.nightlyNext ) {
+		return [
+			await releaseTools.getNextPreRelease( '0.0.0-next' ), null
```

Comment:
I wonder if the `@next` dist tag should follow our nightly pattern or bump a major instead.

Random thoughts:

* `@next` usually means _next version_.
	* Perhaps some developer wants to use it if it does not break anything and helps resolve any issue. 
* We want to deploy it nightly.
* Does the `v45.0.0-next.0` block `v45.0.0-alpha.0`? `next > alpha`, so publishing alpha does not make sense.
* `0.0.0-*` pattern looks like a release you should not use for anything except for testing.

Could we agree to use `@nightly-next` and `0.0.0-nightly-next`?

Reply:
+1 for using `@nightly-next` and `0.0.0-nightly-next`.

Reply:
We agreed to follow our nightly format: `0.0.0-nightly-next-{DATE}`.

Reply:
Addressed in https://github.com/ckeditor/ckeditor5/pull/17884/commits/de1c2cfbcf89731357977794d22bb4b229933039.